docs/110_CustomTemplates.md

```diff
@@ -0,0 +1,184 @@
+# Custom form templates
```

Comment:
Could you please reformat the macros (intent and twig formatter (twig instead of html)

Reply:
or even better: remove the content from the macros 

instead of 

```twig
{% block form_widget_simple -%}
{{ parent() }}
{% if help_text %}
{{ block('help_text_label') }}
{% endif %}
{%- endblock form_widget_simple %}
```

I would suggest

```twig
{% block form_widget_simple -%}
   {# [...] #}
{%- endblock form_widget_simple %}
```